### DISCUSSION TITLE
roadmap of collaborators feature

### DISCUSSION THREAD

**Discussion by ludky** [aircodelabs/aircode#53] [Feature Requests]
你好，尝试使用了一下aircode，非常方便快捷。很期待collaborator的功能。

想询问一下大概ETA会在什么时间段呢？（以后会不会考虑开源让其他developers也可以加入到开发当中呀）
（还有一个问题就是，目前可以share 代码本身，有没有可能增加enable edit之类的功能去软性实现collaboration呢？）

**Comment by adcentury** [comment]
@ludky 你好，非常感谢对于 AirCode 的支持。协作者的功能已经在我们的规划中，预计会在下半年推出，也会考虑结合 GitHub 提供更多样性的协作方式。

关于开源的问题，我们会逐步开源我们系统中的各个部件，包括 Dashboard、SDK 等，欢迎社区开发者为我们的代码进行贡献。
